pkg/loki/flow_query.go

```diff
@@ -120,10 +120,7 @@ func (q *FlowQueryBuilder) addFilter(filter filters.Match) error {
 			q.labelFilters = append(q.labelFilters, lf)
 		}
 	} else if fields.IsIP(filter.Key) {
-		if filter.Not {
-			return fmt.Errorf("'not' operation not allowed in IP filters")
-		}
```

Comment:
@jotak do you remember any reason why not allowing it ?

Reply:
perhaps.. laziness ? Honestly no I don't remember

Reply:
haha no worries. I was wondering if it was related to a specific case not working but I didn't found any during my tests